extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -863,9 +863,9 @@ export class SqlDatabaseTree {
 		this._assessmentTitle.value = selectedIssue?.checkId || '';
 		this._descriptionText.value = selectedIssue?.description || '';
 		this._moreInfo.url = selectedIssue?.helpLink || '';
-		this._moreInfo.label = selectedIssue?.message || '';
+		this._moreInfo.label = selectedIssue?.displayName || '';
 		this._impactedObjects = selectedIssue?.impactedObjects || [];
-		this._recommendationText.value = selectedIssue?.message || ''; //TODO: Expose correct property for recommendation.
+		this._recommendationText.value = selectedIssue?.message || constants.NA;
```

Comment:
`selectedIssue.message` now contains the correct value. The incorrect value that used to be there has now been mapped to `selectedIssue.displayName` instead, so that we can still see it in the 'more info' link's preview text